pkgs/tools/text/chroma/default.nix

```diff
@@ -1,16 +1,32 @@
-{ lib, buildGoModule, fetchFromGitHub }:
+{ lib, buildGoModule, fetchgit, git }:
 
 buildGoModule rec {
   pname = "chroma";
-  version = "0.8.1";
+  version = "0.8.2";
 
-  src = fetchFromGitHub {
-    owner  = "alecthomas";
-    repo   = "chroma";
+  src = fetchgit {
+    url    = "https://github.com/alecthomas/chroma.git";
     rev    = "v${version}";
-    sha256 = "1gwwfn26aipzzvyy466gi6r54ypfy3ylnbi8c4xwch9pkgw16w98";
+    sha256 = "0q14i7gk6vlcfvqzp0h4qz4f27wnskgzshpkk2nw03nirdcjhpsd";
+    leaveDotGit = true;
```

Comment:
Do we really require .git? Can we easily patch it so that it is not required?

Reply:
This is necessary to determine the commit and date of the version which is required to [populate the version number of chroma](https://github.com/alecthomas/chroma/blob/ef102ef33f84f79d119c4e15ebd8bdccff9e1f7f/cmd/chroma/main.go#L85) It would be possible to patch this out however, it'd change the format of the version screen.

Reply:
Why did you switch away from fetchFromGitHub? IIRC it should have also an option to leave the ``.git`` directory.

Reply:
Switched back, has the slight downside that we have to enable `fetchSubmodules` as well to make `fetchFromGitHub` use `fetchgit`.